plugins/grid-bookmark/src/GridBookmarkWidget/components/AssemblySelector.tsx

```diff
@@ -1,114 +1,47 @@
 import { GridBookmarkModel } from '../model'
+import { makeStyles } from 'tss-react/mui'
+import { getSession } from '@jbrowse/core/util'
 
-const useStyles = makeStyles()(() => ({
-  bold: {
-    fontWeight: 700,
+const useStyles = makeStyles()({
+  textfield: {
+    padding: 5,
   },
-}))
+})
 
-const AssemblySelector = observer(function ({
+const GridBookmarkAssemblySelector = observer(function AssemblySelector({
   model,
 }: {
   model: GridBookmarkModel
 }) {
+  const { assemblyManager } = getSession(model)
   const { classes } = useStyles()
-  const { validAssemblies, selectedAssemblies, setSelectedAssemblies } = model
-  const noAssemblies = validAssemblies.length === 0 ? true : false
-  const label = 'Select assemblies'
-  const isAllSelected =
-    validAssemblies.length > 0 &&
-    selectedAssemblies.length === validAssemblies.length
-
-  const handleChange = (event: SelectChangeEvent<string[]>) => {
-    const value = event.target.value
-    if (value.at(-1) === 'all') {
-      setSelectedAssemblies(
-        selectedAssemblies.length === validAssemblies.length
-          ? []
-          : validAssemblies,
-      )
-      return
-    }
-    setSelectedAssemblies([...value])
-  }
-
-  useEffect(() => {
-    // sets the selected assemblies when a valid assembly has been added or removed
-    if (validAssemblies.length > selectedAssemblies.length) {
-      const newAsm = validAssemblies.filter(
-        asm => !selectedAssemblies.includes(asm),
-      )
-      setSelectedAssemblies([...selectedAssemblies, ...newAsm])
-    }
-    if (validAssemblies.length < selectedAssemblies.length) {
-      const rmAsm = selectedAssemblies.filter(
-        asm => !validAssemblies.includes(asm),
-      )
-      rmAsm.forEach(asm => {
-        selectedAssemblies.splice(selectedAssemblies.indexOf(asm), 1)
-      })
-      setSelectedAssemblies([...selectedAssemblies])
-    }
-  }, [
-    validAssemblies.length,
-    selectedAssemblies,
-    setSelectedAssemblies,
-    validAssemblies,
-  ])
+  const { validAssemblies, selectedAssembly = 'SPECIAL_ALL_ASSEMBLIES_VALUE' } =
+    model
 
   return (
-    <FormControl disabled={noAssemblies}>
-      <InputLabel id="select-assemblies-label">{label}</InputLabel>
-      <Select
-        labelId="select-assemblies-label"
-        id="select-assemblies"
-        multiple
-        value={selectedAssemblies}
-        onChange={handleChange}
-        input={<OutlinedInput label={label} />}
-        renderValue={selected => selected.join(', ')}
-      >
-        <MenuItem value="all">
-          <ListItemIcon>
-            <Checkbox
-              checked={isAllSelected}
-              indeterminate={
-                selectedAssemblies.length > 0 &&
-                selectedAssemblies.length < validAssemblies.length
-              }
-            />
-          </ListItemIcon>
-          <ListItemText
-            classes={{ primary: classes.bold }}
-            primary="Select All"
-          />
+    <TextField
```

Comment:
Simplified the multi-select into a single select (via TextField with select on), with an option for "All". The silly SPECIAL_ALL_ASSEMBLY_VALUE could probably be codewise eliminated but wasn't too hard to accomodate

Reply:
I think there's utility to being able to see a subset of assemblies versus all or one, why do you think otherwise?

Reply:
I don't necessarily think otherwise, but I think the UI of multi-selects are complicated, so I tend to avoid.  if you would like to keep, I can restore this function

Reply:
I think it might be helpful to users with a lot of assemblies, especially with bookmarks being persistent conceivably you're going to have a lot more bookmarks at once/more assemblies, and if you're looking at two species at once but have lots of bookmarks in a third you'd have to switch back and forth between the two.